src/cmd/services/m3coordinator/ingest/carbon/ingest_test.go

```diff
@@ -302,6 +333,140 @@ func TestIngesterHonorsPatterns(t *testing.T) {
 	}, found)
 }
 
+func TestIngesterNoStaticRules(t *testing.T) {
+	ctrl := xtest.NewController(t)
+	defer ctrl.Finish()
+
+	mockDownsamplerAndWriter, found := newMockDownsamplerAndWriter(ctrl, func(mappingRules []downsample.AutoMappingRule) {
+		// Use panics instead of require/assert because those don't behave properly when the assertion
+		// is run in a background goroutine.
+		if len(mappingRules) != 1 {
+			panic(fmt.Sprintf("expected: len(DownsampleMappingRules) == 1, got: %v", len(mappingRules)))
```

Comment:
this is going to kill the entire test suite, which seems kind of unfortunate. can we just set a variable that is checked later?

Reply:
That's a better idea. Will fix up